src/plugins/hotkeyPlugin.ts

```diff
@@ -269,8 +272,18 @@ export class HotkeyManager {
   }
 }
 
-const combinationToBindingKey = (combination: HotkeyCombination) => {
-  return combination.toLowerCase().replaceAll(" ", "+");
+/** hotkeys-js用のキーに変換する */
+const combinationToBindingKey = (
+  combination: HotkeyCombination
+): BindingKey => {
+  // MetaキーはCommandキーとして扱う
+  //TODO: Windowsの場合はWindowsキーとして扱う
```

Comment:
一応Windowsでも`Win`キーの入力を認識してくれたので、TODOコメントを書くのは少し違う気がします(書くとしても表示上のMetaをCmdにリプレイスしている場所に書くべきなのかな...?と)。

少しコメントとしては煩雑になりますが、こうなった経緯を丁寧に書いた方が、後のメンテナンスのしやすさは上がるのかなと...

こんな感じだとどうでしょうか...?
もうちょっといい感じにできそうな気もしますが...
```suggestion
  // VOICEVOXでは、ホットキー処理をするためにmousetrapを採用していたが、
  // メンテナンスされていないことからhotkeys-jsに移行した。
  // mousetrapでは、macOSにおけるcommandキー、WindowsにおけるWinキーであるmetaが存在した。
  // しかしhotkeys-jsではmetaの扱いがなく、代わりにcommandがあるので、
  // mousetrapでmetaとして扱われていたキーをcommandに置き換えている。
```

Reply:
> あ、そっちはコメントが既存

なるほど...!まあでもここでなにかできることがあるわけではない(ここを書き換えることで解決することはなさそう)ので、適切でないように感じました。
でもまあ私の感覚なので、TODOコメントのあるなしはどっちでもいいのかなと思います。

> 伝えるべきなのこんな感じですかね

良いと思います！あとからこのコードを見た人がなぜそうなっているのかの経緯がわかりやすくなっているのかなと思いました！

> 前提として個人的にはcmdやwinではなくmetaで保存した方が良い

これは私も同意です。